sci-electronics/iverilog/iverilog-10.3.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="A Verilog simulation and synthesis tool"
+HOMEPAGE="http://iverilog.icarus.com/"
+LICENSE="LGPL-2.1"
+SLOT="0"
+IUSE="examples"
+
+RDEPEND="
+	app-arch/bzip2
+	sys-libs/readline:0=
+	sys-libs/zlib:="
+
+# If you are building from git, you will also need software to generate
+# the configure scripts.
+DEPEND="
+	>=sys-devel/autoconf-2.53
```

Comment:
this too.

Reply:
> this too.

Thank you!

It comes from https://github.com/steveicarus/iverilog/blob/master/README.txt#L73-L76
however it should installed by default with 'inherit autotools'
I removed this dependency.